tests/puppeteer/pages/FO/FO_cart.js

```diff
@@ -0,0 +1,33 @@
+const CommonPage = require('../commonPage');
+
+module.exports = class FO_CART extends CommonPage {
+  constructor(page) {
+    super(page);
+
+    //Selectors for cart page
+    this.productName = '#main li:nth-of-type(%NUMBER) div.product-line-info > a';
+    this.productPrice = '#main li:nth-of-type(%NUMBER) div.current-price > span';
+    this.productQuantity = '#main li:nth-of-type(%NUMBER) div.input-group input.js-cart-line-product-quantity';
+    this.proceedToCheckoutButton = '#main div.checkout a';
+
+    // Selectors for checkout page
+    this.checkoutStepOneTitle = "#checkout-personal-information-step > h1"
+  }
+
+  /**
+   * To check the cart details (product name, price, quantity)
+   */
+  async checkCartDetails(cartData) {
+    await this.checkTextValue(this.productName.replace('%NUMBER', '1'), cartData.name);
```

Comment:
Maybe pass the "1" as a param to the function so that we can target the 2nd or 3rd element without rewriting the function ?